website/docs/components/flyout/partials/guidelines/guidelines.md

```diff
@@ -2,51 +2,111 @@
+![Form within a Flyout](/assets/components/flyout/form-in-flyout.png)
+!!!
+
+#### Actions
 
-![Flyout with a table](/assets/components/flyout/flyout-with-complex-content.png)
+A Flyout supports actions within the footer allowing for basic functions to be performed. This usage should be limited to performing secondary functions that are related to the main page to help the user maintain context in the primary flow. Some examples of this are:
+
+- Batch updating and creating relationships between one or more objects
+- Linking to external resources using a link that appears as a button
+
+!!! Do
+
+Use a Flyout for actions like batch selection that are secondary to the main page.
+
+![Simple actions within a Flyout](/assets/components/flyout/flyout-simple-actions.png)
 !!!
 
-#### Functions within a Flyout
+!!! Do
 
-Given that a Flyout is intended to provide more detail on a specific item, don’t use a Flyout in a functional capacity; e.g., performing a CRUD (create, read, update, delete) function or submitting form data.
+Use a Flyout for simple declarative content like text and links that enhance the main page.
 
-This type of content is often too complex and is better organized in its own page.
+![Complex footer example](/assets/components/flyout/flyout-complex-footer.png)
+!!!
 
 !!! Dont
 
-![Flyout with actions](/assets/components/flyout/flyout-with-form.png)
+Don't use a Flyout for editing or creating objects. These are generally considered primary functions and should be handled on their own page or within a Modal in simple scenarios.
+
+![Complex actions within a Flyout](/assets/components/flyout/flyout-complex-actions.png)
 !!!
 
-#### Code snippets and examples
+### Flyout vs Modal
 
-While code snippets and terminal scripts are usually detailed, they are well suited content within a Flyout due to their contextual relevancy to the content on the page, while maybe not being complex enough to exist on their own page.
+While similar in functionality and interaction, the Flyout and [Modal](/components/modal) are meant to be used in different scenarios and to express different types of content. There is a fair amount of overlap in their usage but they differentiate in these ways:
 
-!!! Do
+#### Complexity
 
-![Flyout with code snippet](/assets/components/flyout/flyout-with-code-snippet.png)
+A Flyout is useful for more complex content given the space it occupies in the viewport, while Modals are useful for less complex content that can be interacted with relatively quickly.
+
+!!! Info
+
+**Note:** Complexity of content is relative, use your own judgement to determine if the content or function is overly complex and consider moving it to its own page.
 !!!
 
+#### Status and messaging
+
+Flyouts are useful when displaying detail content that relates to the page, while Modals are useful for messaging status; e.g., confirming a destructive action or warning about the effects of a change.
```

Comment:
```suggestion
Flyouts are useful when displaying detailed content that relates to the page, while Modals are useful for messaging status; e.g., confirming a destructive action or warning about the effects of a change.
```

Reply:
I might defer to @heatherlarsen on this one, I was intending to use detail as a noun as opposed to a verb. May or may not be correct 🤔

Reply:
Great, thanks both of you for the notes!